blocks/embed/embed.js

```diff
@@ -76,10 +88,17 @@ const loadEmbed = (block, link, autoplay) => {
   ];
 
   const config = EMBEDS_CONFIG.find((e) => e.match.some((match) => link.includes(match)));
-  const url = new URL(link);
+  const url = link ? new URL(link) : '';
+  const blockConfig = readBlockConfig(block);
```

Comment:
readBlockConfig is returning blank config, so we don't find the video id. @andreituicu Any idea why this might be happening?

Reply:
Sorry, I missed this. I'm guessing it's not the case anymore, correct?

Reply:
Unfortunately, it is the case. I tested by hardcoding.